lib/build/bundle.js

```diff
@@ -5,6 +5,7 @@ const Convert = require('./convert-source-map');
 const fs = require('../file-system');
 const SourceInclusion = require('./source-inclusion').SourceInclusion;
 const DependencyInclusion = require('./dependency-inclusion').DependencyInclusion;
+const copyProperty = 'copy';
```

Comment:
Ha ha, gotcha! Yah, I thought that might be what it was. After seeing what you've written, I think using `x.copy` will be just fine :) Thanks.